docs/components/modeler/web-modeler/connectors/available-connectors/sendgrid.md

```diff
@@ -0,0 +1,204 @@
+
+```text
+Subject:
+Your Camunda Cloud Weather Report for {{location}}
+```
+```text
+Body:
+Hi {{name}},
+
+Thanks for using Camunda Cloud Connectors to check your current weather report.
+Your current weather in Berlin is Clouds with {{actual-temp}}°C and feels like {{feel-temp}}°C
+
+The Camunda Cloud Team
+```
+
+In our example template we will use the following Handlebars:
+
+`{{location}}` - the location used for the weather report
+
+`{{name}}` - the name of the user requesting the weather report
+
+`{{actual-temp}}` - the measured temperature
+
+`{{feel-temp}}` - how the temperature feels like in reality
+
+While you are editing your Template you can test how your email would look like by switching to **Preview** mode and choosing **{} Show Test Data** and then providing the necessary data:
+
+![sendgrid email template test](../img/connectors-sendgrid-email-template-test-data.png)
+
+
+#### Create a SendGrid Email Template Connector task
+
+To use this Connector follow the steps below:
+
+1. Open the Web Modeler and create a new BPMN Diagram
+2. Add a **Task** and an **EndEvent**
+3. Click on the **Task**, click on the little spanner/wrench icon, and select **SendGrid Email Template Connector**. You can also use the search field and type `Sendgrid`
+
+![create sendgrid template connector wrench](../img/connectors-sendgrid-create-task-wrench-template.png)
+
+Alternatively, you can directly create a **Sendgrid Email Template Connector** by using the append menu:
+
+![create sendgrid template connector append](../img/connectors-create-task-append.png)
+
+#### Make your SendGrid Email Template Connector executable
+
+To make the **SendGrid Email Template Connector** executable, you need to fill all the mandatory fields that are highlighted in red in the Properties Panel:
+
+![sendgrid email template connector red properties](../img/connectors-sendgrid-email-template-red-properties.png)
+
+
+1. Set **SendGrid API Key** to `secrets.SEND_GRID_API_KEY`
+2. Set **Sender Name** to `Jane Doe` (or the [sender identity](#create-a-sender-identity) you configured above)
+3. Set **Sender Email** to `jane-doe@camunda.com` (or the [sender identity](#create-a-sender-identity) you configured above)
+4. Set **Receiver Name** to `Your Name`
+5. Set **Receiver Email** to `Your email address`
+6. Login to your Sendgrid account and navigate to [the Dynamic Template you created](#configure-a-dynamic-template)
+7. Copy the id of the template and paste it in the **Template ID field**
+
+![get sendgrid template id](../img/connectors-sendgrid-email-template-id.png)
+
+8. Provide the test data in the **Template Data** field
+
+```text
+{
+  location: "Berlin",
+  name: "Jane",
+  actual-temp: 30,
+  feel-temp: 3
+}
+```
+
+
+![sendgrid email template connector complete properties](../img/connectors-sendgrid-email-template-complete-properties.png)
+
+
+:::note
+Now you can simply [deploy and start a new instance](../../save-and-deploy.md) of your process.
```

Comment:
"As with all connectors the run-time is available out of the box in Camunda SaaS 8". ==> I think it is better to unify on our wording here.

What I'd personally love to convey to people is: Connectors are re-usable building blocks that you can use to implement your diagrams. They work (execute) out of the box in Camunda 8 SaaS, no special run-time required.

Reply:
I aligned it with your proposal and also added this box to the rest connector page.